wgpu/gui/qt.py

```diff
@@ -37,25 +37,25 @@ def __init__(self, *args, size=None, title=None, **kwargs):
             self.setWindowTitle(title)
         self.show()
 
-    def paintEngine(self):
+    def paintEngine(self):  # noqa: N802 - this is a Qt method
         # https://doc.qt.io/qt-5/qt.html#WidgetAttribute-enum  WA_PaintOnScreen
         return None
 
-    def paintEvent(self, event):
-        self._drawFrameAndPresent()
+    def paintEvent(self, event):  # noqa: N802 - this is a Qt method
+        self._draw_frame_and_present()
 
-    def getSizeAndPixelRatio(self):
+    def get_size_and_pixel_ratio(self):
```

Comment:
do you think perhaps the entire Qt widget should have Qt style conventions? eh, it's just a little straining on the eyes :) wouldn't mind merging this as is

Reply:
mmm, that's a good question. I think that since the methods that we add here are part of the wgpu API (they come from an abstract base class), they should be snake_case. It actually makes it clear with what API you're interacting with. But yeah, it does not look qt-ish :/